### ISSUE TITLE
Null ref: T2.InputAction.Update_PhaseC_NotDownNorHeld

### ISSUE THREAD

**Issue by ChrisNZL** [ChrisNZL/Tallowmere2#637]
Auto bug report from 0.1.7.

Feedback ID: 20200920-BVBYP

```
11:28:46, Frame 1276, WARNING »  WARNING: GameState_DownloadingLanguage: columnBeingDownloaded: A. webRequest.error: Request timeout

11:29:23, Frame 3458, EXCEPTION »  NullReferenceException: Object reference not set to an instance of an object
>>>>> CRITICAL ERROR >>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>

T2.InputAction.Update_PhaseC_NotDownNorHeld (T2.GameState newGameState)
T2.InputAction.Update_PhaseB_Allowed (T2.GameState newGameState)
T2.InputAction.Update ()

SystemPlayer InputDevice: Keyboard
```

**Comment by ChrisNZL** [comment]
For 0.1.8, added additional null checks within `Update_PhaseC_NotDownNorHeld`.

If issue persists, will need to break method into smaller methods.

**Comment by ChrisNZL** [comment]
Still noticeable at startup – seems to be an issue when testing translations via Google Sheets.

Screen remains blank if failing to download a language sheet:
- `WARNING: GameState_DownloadingLanguage: columnBeingDownloaded: A. webRequest.error: HTTP/1.1 403 Forbidden`

In-game red top "Critical Error" panel appears, but pressing F2 does not show the Debug Log.

```
8:42:48, Frame 679, WARNING »  WARNING: GameState_DownloadingLanguage: columnBeingDownloaded: A. webRequest.error: HTTP/1.1 403 Forbidden

8:42:55, Frame 1120, EXCEPTION »  NullReferenceException: Object reference not set to an instance of an object
>>>>> CRITICAL ERROR >>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>

T2.InputAction.Update_PhaseC_NotDownNorHeld (T2.GameState newGameState)
T2.InputAction.Update_PhaseB_Allowed (T2.GameState newGameState)
T2.InputAction.Update ()

SystemPlayer InputDevice: Keyboard
```

**Comment by ChrisNZL** [comment]
Should be fixed in 0.1.8.

Added additional null checks, along with better handling when starting up and testing language spreadsheet importing.
